tests/test_query.py

```diff
@@ -210,15 +210,20 @@ def test_post_query_invalid(client: ThanoSQL, basic_table_name: str):
 
 def test_post_query_success(client: ThanoSQL, basic_table_name, empty_table_name):
     params = {"table_name": basic_table_name, "columns": query_test_selected_columns}
-    completed_changed_query = f"SELECT name, price FROM {basic_table_name}"
+    completed_changed_query = f"SELECT name, price FROM {basic_table_name} LIMIT 0"
 
     # direct entry template
-    res = client.query.execute(query=changed_query_template_string, parameters=params)
+    res = client.query.execute(
+        query=changed_query_template_string, parameters=params, max_results=10
+    )
     assert isinstance(res, QueryLog)
     assert res.error_result is None
     assert res.destination_schema == "qm"
     assert res.query == completed_changed_query
 
+    # there should be no records even if max_results is > 0 as we set LIMIT 0 in the query
```

Comment:
```suggestion
    # there should be empty record([]) even if max_results is > 0 as we set LIMIT 0 in the query
```
Below, even if it is None, it is expressed as 'no records', which is confusing.